types/styled-components/test/index.tsx

```diff
@@ -1185,3 +1187,11 @@ function unionTest2() {
     <C />; // $ExpectError
     <C foo={123} bar="foobar" />; // $ExpectError
 }
+
+function createCustomGlobalStyle(style: string): GlobalStyleComponent<null, DefaultTheme> {
+    return createGlobalStyle`
+        ${style}
+        * {
+        box-sizing: border-box;
+    }`;
+}
```

Comment:
This test is passing on `master`

Reply:
Excuse me, I do not know why it is, but I had an error in my environment.

I did following commands.

```
git checkout master
git cherry-pick 5b75a6defaa76e9d93d12f8623dbc7416a7e213d
npm run test storybook
```

I received this error.

```
ERROR: 1191:5  expect  TypeScript@4.3 compile error:
Type 'GlobalStyleComponent<object, DefaultTheme>' is not assignable to type 'GlobalStyleComponent<null, DefaultTheme>'.
  Types of property 'propTypes' are incompatible.
    Type 'WeakValidationMap<ThemedGlobalStyledClassProps<object, DefaultTheme>> | undefined' is not assignable to type 'undefined'.
      Type 'WeakValidationMap<ThemedGlobalStyledClassProps<object, DefaultTheme>>' is not assignable to type 'undefined'.
```

Reply:
We should remove this test if it's unrelated to the change. If you think it adds coverage, we can add it in a follow-up.